tools/postgres-init-dev-db

```diff
@@ -24,7 +24,7 @@ if [[ $# == 0 ]]; then
     USERNAME=zulip
     PASSWORD=$("$(dirname "$0")/../bin/get-django-setting" LOCAL_DATABASE_PASSWORD)
     DBNAME=zulip
-    SEARCH_PATH="$USERNAME",public
+    SEARCH_PATH="$USERNAME",public,pgroonga,pg_catalog
```

Comment:
> OK, makes sense. Can you add a comment explaining this in the relevant section of zproject/settings.py?

Done.

> Also, you'll need to put this in scripts/setup/postgres-init-db as well (the version that runs for setting up production instances).

We should not add `pgroonga` to search when we don't use PGroonga. If PGroonga is installed but Zulip doesn't use PGroonga, `@@` operator in `pgroonga` schema is used. It's a problem.

We need to know whether does the user want to use PGroonga or not in `scripts/setup/postgres-init-db`. If the user wants to use PGroonga, we should setup search path. How do we know about it?